src/utils.ts

```diff
@@ -593,68 +592,74 @@ function cleanIframes() {
   }
 }
 
-export function linkFrameWindow(frame: HTMLIFrameElement) {
+export function linkFrameWindow(frame: HTMLIFrameElement): void {
   cleanIframes();
 
   if (frame && frame.contentWindow) {
     try {
       iframeWindows.push(frame.contentWindow);
       iframeFrames.push(frame);
-    } catch (err) {
+    } catch (err: unknown) {
       // pass
     }
   }
 }
 
-export function getUserAgent(win: ?SameDomainWindowType): string {
+export function getUserAgent(
+  win: SameDomainWindowType | undefined | undefined
```

Comment:
Does `undefined | undefined` serve some purpose, or is this just a weird artifact from the flow-to-ts tool?  Looks like it's present in a few places.

Reply:
Good catch @dtjones404! I assume we should only have one `undefined`